2024-04-04 18:15:04,015 INFO    MainThread:58421 [wandb_setup.py:_flush():76] Current SDK version is 0.16.6
2024-04-04 18:15:04,015 INFO    MainThread:58421 [wandb_setup.py:_flush():76] Configure stats pid to 58421
2024-04-04 18:15:04,015 INFO    MainThread:58421 [wandb_setup.py:_flush():76] Loading settings from /Users/aida/.config/wandb/settings
2024-04-04 18:15:04,015 INFO    MainThread:58421 [wandb_setup.py:_flush():76] Loading settings from /Users/aida/PycharmProjects/ec523-project/ec523-project/wandb/settings
2024-04-04 18:15:04,015 INFO    MainThread:58421 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-04-04 18:15:04,015 INFO    MainThread:58421 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-04-04 18:15:04,015 INFO    MainThread:58421 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'main.py', 'program_abspath': '/Users/aida/PycharmProjects/ec523-project/ec523-project/main.py', 'program': '/Users/aida/PycharmProjects/ec523-project/ec523-project/main.py'}
2024-04-04 18:15:04,015 INFO    MainThread:58421 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-04-04 18:15:04,016 INFO    MainThread:58421 [wandb_init.py:_log_setup():521] Logging user logs to /Users/aida/PycharmProjects/ec523-project/ec523-project/wandb/run-20240404_181504-t5crmt25/logs/debug.log
2024-04-04 18:15:04,016 INFO    MainThread:58421 [wandb_init.py:_log_setup():522] Logging internal logs to /Users/aida/PycharmProjects/ec523-project/ec523-project/wandb/run-20240404_181504-t5crmt25/logs/debug-internal.log
2024-04-04 18:15:04,016 INFO    MainThread:58421 [wandb_init.py:init():561] calling init triggers
2024-04-04 18:15:04,016 INFO    MainThread:58421 [wandb_init.py:init():568] wandb.init called with sweep_config: {}
config: {'exp_name': 'main', 'alg_name': 'DQN', 'seed': 1, 'torch_deterministic': True, 'cuda': True, 'track': True, 'wandb_project_name': 'ec523', 'wandb_entity': None, 'capture_video': True, 'save_model': False, 'upload_model': False, 'env_id': 'CartPole-v1', 'total_timesteps': 500000, 'learning_rate': 0.00025, 'num_envs': 1, 'buffer_size': 10000, 'gamma': 0.99, 'tau': 1.0, 'target_network_frequency': 500, 'batch_size': 128, 'start_e': 1, 'end_e': 0.05, 'exploration_fraction': 0.5, 'learning_starts': 10000, 'train_frequency': 10}
2024-04-04 18:15:04,016 INFO    MainThread:58421 [wandb_init.py:init():611] starting backend
2024-04-04 18:15:04,016 INFO    MainThread:58421 [wandb_init.py:init():615] setting up manager
2024-04-04 18:15:04,019 INFO    MainThread:58421 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-04-04 18:15:04,021 INFO    MainThread:58421 [wandb_init.py:init():623] backend started and connected
2024-04-04 18:15:04,028 INFO    MainThread:58421 [wandb_init.py:init():715] updated telemetry
2024-04-04 18:15:04,055 INFO    MainThread:58421 [wandb_init.py:init():748] communicating run to backend with 90.0 second timeout
2024-04-04 18:15:04,400 INFO    MainThread:58421 [wandb_run.py:_on_init():2357] communicating current version
2024-04-04 18:15:04,570 INFO    MainThread:58421 [wandb_run.py:_on_init():2366] got version response 
2024-04-04 18:15:04,571 INFO    MainThread:58421 [wandb_init.py:init():799] starting run threads in backend
2024-04-04 18:15:04,858 INFO    MainThread:58421 [wandb_run.py:_console_start():2335] atexit reg
2024-04-04 18:15:04,858 INFO    MainThread:58421 [wandb_run.py:_redirect():2190] redirect: wrap_raw
2024-04-04 18:15:04,858 INFO    MainThread:58421 [wandb_run.py:_redirect():2255] Wrapping output streams.
2024-04-04 18:15:04,858 INFO    MainThread:58421 [wandb_run.py:_redirect():2280] Redirects installed.
2024-04-04 18:15:04,859 INFO    MainThread:58421 [wandb_init.py:init():842] run started, returning control to user process
2024-04-04 18:15:04,860 INFO    MainThread:58421 [wandb_run.py:_tensorboard_callback():1505] tensorboard callback: runs/DQN_CartPole-v1_default_network_1_1712268902, True
2024-04-04 18:19:53,236 WARNING MsgRouterThr:58421 [router.py:message_loop():77] message_loop has been closed
